### DISCUSSION TITLE
Runing xiSearch1.8.7 without no results

### DISCUSSION THREAD

**Discussion by joe11chen** [Rappsilber-Laboratory/XiSearch#124] [Q&A]
Dear xiSearch team,

I was using xiSearch 1.8.7 to process my BS3 cross-link data (parameters is shown in the figure 1 below). I saw the message "100% processed"(as shown in the figure 2 below) in the feedback log area, but there was no "completed" message in the log, nor any output in the result CSV file. I am wondering if something went wrong.
![image](https://github.com/user-attachments/assets/b3930f58-c8bc-429e-b9d6-82099829f30d)
![image](https://github.com/user-attachments/assets/c0921fae-f02c-4fe4-b5ce-fd634011ca4b)

Best regards,
JoeChen

**Comment by grandrea** [comment]
hi i am sorry I missed this. Did you manage to resolve? Could you share how the "files" window looks like?

**Comment by lutzfischer** [comment]
If this happen again can you post the log file - should be in the same folder as the result file.
